usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -19,6 +20,111 @@
 ========
 """
 
+"""@pylatest api/gluster.cluster_create
+API-gluster: cluster_create
+***************************
+
+.. test_metadata:: author fbalak@redhat.com
+
+Description
+===========
+
+Positive create gluster cluster.
+"""
+
+
+def test_cluster_create_valid(
+        valid_session_credentials,
+        valid_nodes,
+        network_interface):
+    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+    """@pylatest api/gluster.cluster_import
+        .. test_step:: 1
+
+            Send POST request to Tendrl API ``APIURL/GlusterCreateCluster
+
+        .. test_result:: 1
+
+            Server should return response in JSON format:
+
+                {
+                  "job_id": job_id
+                }
+
+            Return code should be **202**
+                with data ``{"message": "Accepted"}``.
+
+        """
+    nodes = []
+    provisioner_ip = None
+    network = None
+    node_ids = []
+    for x in valid_nodes:
+        if "tendrl/server" in x["tags"]:
+            network = "{}/22".format(
+                ast.literal_eval(x["networks"][network_interface]["ipv4"])[0])
+            continue
+        ips = ast.literal_eval(x["networks"][network_interface]["ipv4"])
+        nodes.append({
+            "role": "glusterfs/node",
+            "ip": ips[0] if type(ips) == list else ips})
+        node_ids.append(x["node_id"])
+        if "provisioner/gluster" in x["tags"]:
+            provisioner_ip = ips[0] if type(ips) == list else ips
+    LOGGER.debug("node_ips: %s" % nodes)
+    LOGGER.debug("provisioner: %s" % provisioner_ip)
+    """@pylatest api/gluster.cluster_create
+        .. test_step:: 2
+
+        Check if there is at least one gluster node for cluster creation.
+
+        .. test_result:: 2
+
+        Test passes if there is at least one gluster node.
+        """
+    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+    pytest.check(
+        len(nodes) > 0,
+        "There have to be at least one gluster node."
+        "There are {}".format(len(valid_nodes)))
+    job_id = api.create_cluster(
+        "MyCluster",
+        "4654ac00-e67b-4b74-86a3-e740b1b8cee5",
+        nodes,
+        provisioner_ip,
+        network)["job_id"]
+
+    api.wait_for_job_status(job_id)
+
+    integration_id = api.get_job_attribute(
+        job_id=job_id,
+        attribute="TendrlContext.integration_id",
+        section="parameters")
+    LOGGER.debug("integration_id: %s" % integration_id)
+
+    # TODO(fbalak) remove this sleep after
```

Comment:
is there issue for this workaround. And also please move this workaround to separated PR.